src/test/java/com/myMoneyTracker/dao/AppUserDaoTest.java

```diff
@@ -38,14 +38,6 @@ public void shouldSaveAppUser(){
         assertTrue(appUser.getId() != 0);
```

Comment:
Modified the place of non-test method. All non-test methods should be after the test methods